Source/Core/InputCommon/InputConfig.cpp

```diff
@@ -117,3 +117,17 @@ bool InputConfig::ControllersNeedToBeCreated() const
 {
 	return m_controllers.empty();
 }
+
+bool InputConfig::IsControllerAKeyboard(int index) const
```

Comment:
I had problems thinking of a better name that wasn't too wordy. Suggestions welcome.